### PR TITLE
Switch middleware to the items module (fix errors)

### PR DESCRIPTION
**Pull request by charypar**
Fixes errors raised by the nav model trying to parse a json file that no longer exists.

### CODE REVIEW

**Comment by charypar** [comment]
Hopefully this doesn't break anything else. I feel like I have no idea what I'm doing in here :)

File: gulpfile.js [hunk 1]
```diff
@@ -9,11 +9,27 @@ var topLevelSections = ['world', 'companies', 'markets', 'opinion', 'work & care
...
+
+	middleware(mockReq, mockRes, function() {
+		if(!mockRes.locals.nav || mockRes.locals.nav.length < 1) {
+			throw "The nav middleware doesn't set navigation config!";
+		}
+	});
+
+	if(!mockRes.locals.nav || mockRes.locals.nav.length < 1) {
+		throw "The nav middleware doesn't set navigation config!";
+	}
```
Comment (ironsidevsquincy) [inline gulpfile.js]: what's this one for?

**Comment by ironsidevsquincy** [comment]
:+1:

File: gulpfile.js (hunk 1 above)
Comment (charypar) [inline gulpfile.js]: In case the `next` method doesn't get called. Actually, that should be a check that it did. Let me fix. I must be still asleep...

**Comment by matthew-andrews** [comment]
thanks viktor !

**Pull request merged**